src/components/CreateGameDialog.vue

```diff
@@ -1,61 +1,62 @@
 <template>
-  <v-dialog v-model="show">
-    <template #activator="{ props }">
-      <v-btn v-bind="props" color="primary" data-cy="create-game-btn" rounded elevation="8">
+  <base-dialog v-model="show" id="create-game" title="Create Game">
+    <template #activator>
+      <v-btn class="text-surface-2" color="surface-1" data-cy="create-game-btn" rounded elevation="8">
         Create Game
       </v-btn>
     </template>
-    <v-card data-cy="create-game-dialog">
-      <v-card-title>
-        <h2>Create Game</h2>
-      </v-card-title>
-      <v-card-text>
-        <div class="d-flex align-center">
-          <v-switch v-model="isRanked" label="Ranked" data-cy="create-game-ranked-switch" color="primary" />
-          <stats-scoring-dialog :show-button-text="false" />
-        </div>
-        <v-text-field
-          v-model="gameName"
-          autofocus
-          :disabled="loading"
-          label="Game Name"
-          variant="outlined"
-          data-cy="game-name-input"
-          @keydown.enter="submitNewGame"
+    <template #body>
+      <div class="d-flex align-center">
```

Comment:
Let's make this a `<form name="create_game_form">` and then use a `<button type="submit" form="create_game_form">` in the `#actions` to do the submit. That way we've got more semantic html and the use of the `name` and `form` attributes can make the form submit work even though the buttons in the `#actions` slot which is outside the `<form>` itself